<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tic-Tac-Toe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button1_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Shortcut[1]</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Shortcut[2]</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Shortcut[3]</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Shortcut[6]</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="shortcut">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Shortcut[4]</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Shortcut[5]</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="shortcut">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Shortcut[8]</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="shortcut">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Shortcut[7]</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="shortcut">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Shortcut[9]</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayerX">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/graphics/XPlayer.svg</normaloff>:/graphics/XPlayer.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayerY">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/graphics/OPlayer.svg</normaloff>:/graphics/OPlayer.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="playerX">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player X</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerO">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player O</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>580</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabelX">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>55</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabelO">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>55</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SCORE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_game"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_game">
   <property name="text">
    <string>New game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quit the application&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
